app/javascript/app/components/nav/nav-component.jsx

```diff
@@ -4,31 +4,48 @@ import Link from 'redux-first-router-link';
 import cx from 'classnames';
 
 import Icon from 'components/icon';
+import NavLinks from 'components/nav-links';
 
 import cwLogo from 'assets/cw-logo.svg';
+import navBarTheme from 'styles/themes/nav-links/nav-links-nav-bar.scss';
 import styles from './nav-styles.scss';
 
 class Nav extends PureComponent {
   render() {
-    const { routes, className } = this.props;
+    const { routes, actions, className } = this.props;
     return (
-      <nav className={cx(styles.navbar, className)}>
-        <Link className={styles.link} to="/">
-          <Icon className={styles.logo} icon={cwLogo} />
-        </Link>
-        {routes.map(route => (
-          <Link className={styles.link} key={route.label} to={route.path}>
-            {route.label}
-          </Link>
-        ))}
-      </nav>
+      <div className={styles.navbarContainer}>
+        <nav className={cx(styles.navbar, className)}>
+          <div className={cx(styles.navMenu)}>
+            <Link className={styles.logoContainer} to="/">
```

Comment:
Does the app logo redirects to any `path`, for what I've understood by the designs the _home_ page is `/planning`, so maybe we do not need to use a `Link`for the logo.